generators/readme/README.md

```diff
@@ -0,0 +1,19 @@
+# Readme generator
+
+A package to generate README files for the React component packages.
+
+## Prerequisites
+
+Each package should have the following fragment files in a `docs` folder:
+
+- `docs/usage-example.js`: a JS example for using the component.
+- `docs/properties.md`: the markdown content for the describing the component's properties (usually as a table).
+- `docs/additional-info.md`: any additional information to be rendered AFTER the generated sections.
```

Comment:
Where would the basic info go. I mean the intro/description. Keep in mind that not all of the components has this description section but of them has.